examples/emodel_pipeline_local_python/export.sbatch

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#####################################################################
+# Copyright 2023, EPFL/Blue Brain Project
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#####################################################################
+
+#SBATCH --account=proj000
+#SBATCH --error=./logs/export_%j.log
+#SBATCH --output=./logs/export_%j.log
+#SBATCH --ntasks=10
+#SBATCH --time=03:00:00
+#SBATCH --constraint=cpu
+#SBATCH --partition=prod
+
+set -e
+set -x
+
+export IPYTHON_PROFILE=export_${SLURM_JOB_ID}_$(hostname)
```

Comment:
export_emodels_hoc (unlike export_emodels_sonata) does not run the cells. So it should be a very quick to run function, and I don't think we need ipyparallel for that, it is kind of an overkill